src/apps/core/views/monster-key-value-editor.html

```diff
@@ -0,0 +1,4 @@
+<div class="monster-key-value-editor">
+	<div class="monster-key-value-data-container"></div>
+	<a class="monster-link" data-action="add">{{coalesce addLink i18n.keyValueEditor.addRow}}</a>
```

Comment:
```suggestion
	<a class="monster-link" data-action="add">
		{{coalesce addLink i18n.keyValueEditor.addRow}}
	</a>
```
Same here, the `data-action` is not necessary and and `id` tag can be used instead for better selector targeting.